lib/inferno/config/boot/ig_files.rb

```diff
@@ -0,0 +1,29 @@
+require 'fileutils'
+
+Inferno::Application.register_provider(:ig_files) do
+  prepare do
+    # This process should only run once, so skipping it on workers will start it
+    # only once from the "web" process
+    next if Sidekiq.server?
+
+    target_container.start :logging
+
+    test_kit_gems = Bundler.definition.specs.select { |spec| spec.metadata['inferno_test_kit']&.casecmp? 'true' }
+
+    ig_files = test_kit_gems.flat_map do |test_kit|
+      Dir.glob(File.join(test_kit.full_gem_path, 'lib', '*', 'igs', '*.tgz'))
+    end
+
+    if test_kit_gems.none? { |gem| gem.full_gem_path == Dir.pwd }
+      ig_files += Dir.glob(File.join(Dir.pwd, 'lib', '*', 'igs', '*.tgz'))
+    end
```

Comment:
This is to handle test kits that do not have that metadata?

Reply:
Or whatever the current repo is which may not be a test kit (platform, inferno core).

Reply:
Oh ok.  Sounds good.

I guess i'd prefer a rule that is just "platforms don't provide these things or attempt to override test kits, if you try you are doing it wrong"... but i suppose for core that makes sense.  I dislike how core is a bit of a strange hybrid-test kit thingy... why not just have core include a 'normal' test kit inside a subdir it that is launched for testing, instead of maintaining a kinda-test-kit-thing overlayed on the same file path structure as core?  Oh well.